src/store/messagesStore.js

```diff
@@ -1313,6 +1334,18 @@ const actions = {
 	async easeMessageList(context, { token }) {
 		context.commit('easeMessageList', { token })
 	},
+
+	async getTranslationConfiguration(context) {
+		try {
+			const response = await getTranslationLanguages()
```

Comment:
We populate this via initial state, so you could save the request (and we have a feature toggle already ready?)

Reply:
hmm okay wouldn't work for desktop unless that loads it itself?

Reply:
Not aware about toggle, but yes, restore it as initial, getting from capabilities